docs/api.md

```diff
@@ -1,21 +1,21 @@
 - [Cloud Hypervisor API](#cloud-hypervisor-api)
-  * [External API](#external-api)
-    + [REST API](#rest-api)
-      - [Location and availability](#location-and-availability)
-      - [Endpoints](#endpoints)
-		* [Virtual Machine Manager (VMM) Actions](#virtual-machine-manager-vmm-actions)
-		* [Virtual Machine (VM) Actions](#virtual-machine-vm-actions)
-      - [REST API Examples](#rest-api-examples)
-        * [Create a Virtual Machine](#create-a-virtual-machine)
-	    * [Boot a Virtual Machine](#boot-a-virtual-machine)
-        * [Dump a Virtual Machine Information](#dump-a-virtual-machine-information)
-        * [Reboot a Virtual Machine](#reboot-a-virtual-machine)
-        * [Shut a Virtual Machine Down](#shut-a-virtual-machine-down)
-    + [Command Line Interface](#command-line-interface)
-    + [REST API and CLI Architecural Relationship](#rest-api-and-cli-architectural-relationship)
-  * [Internal API](#internal-api)
-    + [Goals and Design](#goals-and-design)
-  * [End to End Example](#end-to-end-example)
+  - [External API](#external-api)
+    - [REST API](#rest-api)
+    - [Location and availability](#location-and-availability)
+    - [Endpoints](#endpoints)
+      - [Virtual Machine Manager (VMM) Actions](#virtual-machine-manager-vmm-actions)
+      - [Virtual Machine (VM) Actions](#virtual-machine-vm-actions)
+    - [REST API Examples](#rest-api-examples)
+      - [Create a Virtual Machine](#create-a-virtual-machine)
+      - [Boot a Virtual Machine](#boot-a-virtual-machine)
+      - [Dump a Virtual Machine Information](#dump-a-virtual-machine-information)
+      - [Reboot a Virtual Machine](#reboot-a-virtual-machine)
+      - [Shut a Virtual Machine Down](#shut-a-virtual-machine-down)
+    - [Command Line Interface](#command-line-interface)
+    - [REST API and CLI Architectural Relationship](#rest-api-and-cli-architectural-relationship)
+  - [Internal API](#internal-api)
+    - [Goals and Design](#goals-and-design)
+  - [End to End Example](#end-to-end-example)
```

Comment:
Are all these changes made on purpose? Just checking because this looks like it could have been made by your editor directly.
Also, if they're intended, I would add them in their own commit.

Reply:
No idea. Well spotted, i'll keep an eye out for it again.